tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/ClassEmitter.scala

```diff
@@ -337,11 +340,12 @@ private[emitter] final class ClassEmitter(jsGen: JSGen) {
   }
 
   /** Generates the creation of fields for a Scala class. */
-  private def genFieldDefsOfScalaClass(tree: LinkedClass)(
+  private def genFieldDefsOfScalaClass(tree: LinkedClass,
+      fields: List[FieldDef])(
```

Comment:
Remove `tree` from args, it is unused (`tpe` is also unused). As a consequence, probably rename the method.

Reply:
Well spotted. I would keep the name, though, as generating fields for a JS class is done elsewhere, and doesn't do the same thing.